Makefile

```diff
@@ -1,3 +1,7 @@
 .PHONY: generate
 generate:
 	$(MAKE) -C resources/mixins/kubernetes generate
+
+.PHONY: update
+update:
+	$(MAKE) -C resources/mixins/kubernetes update
```

Comment:
Who will call it? How about adding a workflow that will handle that weekly?

Reply:
We don't really need to update the mixins very frequently, only if we want a new feature of a newer version. So I think keeping it at manual casual updates is good enough.